docs/docs/api/param.md

```diff
@@ -6,14 +6,17 @@ sidebar_label: Param
 
 `Param` allow using runtime values from the view/producer to create new paths.
 
-Occasionally, a `producer` (`view`s are also producers) need to access a path in
-state, which depends on a value which is created in the producer itself. `Param`
-is the path-composition operator to use in such sitations.
+Occasionally, a `producer` or `view` need to access a path in state, which
+depends on a value which is created in the producer itself. `Param` is the
+[path-composition](/docs/concepts/path-composition) operator to use in such
+situations.
 
 For example, `Update.*` functions can be given an object in their second
 argument. Every key in this object can then be referred to with `Param.<key>` in
 the header
 
+## Example
```

Comment:
Maybe we could label these sections (in all keywords pages) as `Usage` instead of `Example`.

Being the API section, I guess the code presented needs to accompany the definition (introduction) of that keyword. In order words, the code snippet should be a comprehensive overview of how that keyword is used and not a true example of running code.